discover/faq.md

```diff
@@ -12,1638 +12,164 @@ Hopefully, you'll find the answers here! If not then please don't hesitate to re
 ### General
 
 <details>
+<summary>How is Ocean Protocol related to artificial intelligence? </summary>
```

Comment:
You can easier review this file by going to: https://github.com/oceanprotocol/docs/blob/29aab5963722b6d521eb6f02830c34cc60f329ba/discover/faq.md(View file menu)